okhttp/src/main/java/okhttp3/OkHttpClient.kt

```diff
@@ -1025,6 +1034,17 @@ open class OkHttpClient internal constructor(
       pingInterval(duration.toMillis(), MILLISECONDS)
     }
 
+    /**
+     * Sets minimum outbound web socket message size (in bytes) that will be compressed.
+     *
+     * Set to 0 to enable compression for all outbound messages.
+     *
+     * [RealWebSocket.DEFAULT_MINIMUM_DEFLATE_SIZE] by default.
```

Comment:
Probably shouldn't refer to an internal class for documentation, since it will encourage use, alternatively we may want this default to be accessible.

Reply:
Done. There are now 3 copies of the 1024 value in plain text though.